<launch>
        <!-- Config -->
        <arg name="robot_base_link" default="base_link"/>
        <arg name="end_effector_link" default="tool0"/>
        <arg name="moving_tf_frame" default="follow_me"/>
        <arg name="target_frame_topic" default="/my_cartesian_motion_controller/target_frame"/>
        <arg name="publish_rate" default="100"/>

        <!-- Turn a TF frame into a moving target for CartesianMotionControllers -->
        <node name="moving_target" pkg="cartesian_controller_handles" type="moving_target.py" output="screen">
                <param name="robot_base_link" value="$(arg robot_base_link)" />
                <param name="end_effector_link" value="$(arg end_effector_link)" />
                <param name="moving_tf_frame" value="$(arg moving_tf_frame)" />
                <param name="target_frame_topic" value="$(arg target_frame_topic)" />
                <param name="publish_rate" value="$(arg publish_rate)" />
        </node>
</launch>
